src/common/PersonTile/styled.js

```diff
@@ -69,4 +70,52 @@ export const Name = styled.div`
   @media (max-width: ${({ theme }) => theme.breakpoint.mobileMin}px) {
     font-size: 14px;
   }
+  ${({ list }) =>
+    list &&
+    css`
+      -webkit-line-clamp: 2;
+      max-height: 2.6em;
+    `}
 `;
+
+
+export const Person = styled.article`
+  padding: 16px;
+  background-color: ${({ theme }) => theme.color.white};
```

Comment:
you missed the "s" :) please replace with : color: ${({ theme }) => theme.colors.white} 👍